projects/plugins/boost/app/modules/cache/file_cache.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+require_once __DIR__ . '/cache.php';
+
+class Boost_File_Cache extends Boost_Cache {
```

Comment:
The functions and members of this class are undocumented; please add docblocks.

Reply:
Done